README.md

```diff
@@ -203,7 +203,11 @@ These are activated by having an environment variable FEATURES_(flag name) set t
 FEATURES_rate_limiting=active bundle exec rails s
 ```
 
-The available flags are listed in `app/services/feature_flag.rb`, and available in the constant `FeatureFlag::FEATURES`. Each one is tested with a dedicated spec in `spec/features/feature_flags/`.
+The available flags are listed in `app/services/feature_flag.rb`, and available in the constant `FeatureFlag::FEATURES`.
+
+Name                           | Description
+-------------------------------| -----------------------------------------------------------------------------------------------------------------------------------
+FEATURES_gias_data_stage_pause | Pauses any GIAS data staging of trusts and schools, should be used e.g. in Septemeber when GIAS data is changed a lot and unsettled
 
```

Comment:
Adding this here to explain what it does, we should do that with flags in the future to document